models/news-models.js

```diff
@@ -26,6 +26,10 @@ exports.fetchAllArticles = (sort_by = "created_at", order = "desc", topic) => {
         return Promise.reject({status: 400, msg: 'Invalid order query'})
     }
 
+    if (topic && (!String(topic) || !isNaN(topic))) {
```

Comment:
if the topic passed is invalid PSQL will throw an error that you can catch and handle in the app rather than writing a whole new custom error